packages/zoe/src/contracts/loan/borrow.js

```diff
@@ -15,14 +15,22 @@ import { makeAddCollateralInvitation } from './addCollateral';
 /** @type {MakeBorrowInvitation} */
 export const makeBorrowInvitation = (zcf, config) => {
   const {
-    mmr, // Maintenance Margin Requirement, in percent
+    mmr, // Maintenance Margin Requirement, as a Percent (deprecated)
```

Comment:
Is there a reason not to change this now? I think nothing depends on the loan using a mmr percent. Seems like more work and more confusing to keep as a percent